plugin/description.html

```diff
@@ -1 +1 @@
-Rust language support
+The plugin provides support for Rust language
```

Comment:
It's a hotfix than real update of the description

@KseniaStar if you think we need a more detailed description (I personally think it can be useful), let's update it separately

Reply:
@Undin HI! I also think we should extend the description a bit. How about this:
`The plugin adds support for Rust language. Enjoy the full IDE experience with smart coding assistance, seamless Cargo support, a fully-fledged Rust debugger, easy access to remote environments, code coverage tooling, and more.`
What do you think?